livre/lib/data/MemoryDataSource.cpp

```diff
@@ -88,21 +88,27 @@ MemoryUnitPtr MemoryDataSource::getData( const LODNode& node )
                             _volumeInfo.getBytesPerVoxel();
     const Identifier nodeID = node.getNodeId().getId();
     const uint8_t* id = reinterpret_cast< const uint8_t* >( &nodeID );
-    const uint8_t fillValue = ( id[0] ^ id[1] ^ id[2] ^ id[3] ) + 16 +
+    const uint8_t opacity =  ( id[0] ^ id[1] ^ id[2] ^ id[3] ) + 16 +
```

Comment:
I think that intensity is a valid and commonly accepted name:
- https://www.google.com/search?client=ubuntu&channel=fs&q=pixel+intensity+definition&ie=utf-8&oe=utf-8
- https://www.google.com/search?client=ubuntu&channel=fs&q=pixel+intensity+definition&ie=utf-8&oe=utf-8#channel=fs&q=voxel+intensity